packages/carbon-react/.storybook/styles.scss

```
@@ -22,3 +46,16 @@ body {
 [data-carbon-theme='g100'] {
   @include styles.theme(styles.$g100);
 }
+
+html[lang='en'] body {
+  font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
```

I think with our styles we might be able to do:

```suggestion
  font-family: styles.font-family('sans');
```

Totally get if it doesn't work now though, but we should definitely add it in through the `scss/_type.scss` entrypoint if not

---

Should we include this via `@ibm/plex` or in the `@carbon/type`? Might be good to create a separate issue to track this enhancement as it doesn't currently exist

---

@tw15egan ooo good call, carbon type would totally be the way to go for this. Definitely get wanting to get this PR in and then we can tackle getting in carbon type as an enhancement 👍